Topic 0/src/com/globant/topic0/exercise2/CountryHouse.java

```diff
@@ -0,0 +1,48 @@
+package com.globant.topic0.exercise2;
+
+public class CountryHouse implements HouseBuilder {
```

Comment:
The pattern is not correct, since you only can build a house with a surface=200, rooms=8.... 
So you need to change all the interface to accept arguments in the methods, and be reliable to make any house that you want.